libvast/src/data.cpp

```diff
@@ -518,6 +519,10 @@ bool convert(const caf::config_value& x, data& y) {
   return caf::visit(f, x);
 }
 
+std::string to_string(const data& d) {
+  return fmt::format("{:a}", d);
+}
```

Comment:
We can probably just add a `using fmt::to_string` somewhere in the namespace `vast`. Maybe in `fwd.hpp`.

Reply:
Adding this works bad with concept/printable that is still there.

Reply:
Yes, this will have to wait until printable is removed entirely.